polaris-react/src/components/EmptyState/EmptyState.tsx

```diff
@@ -90,56 +92,61 @@ export function EmptyState({
     : null;
 
   const headingMarkup = heading ? (
-    <Text variant={headingSize} as="p">
-      {heading}
-    </Text>
+    <Box paddingBlockEnd="4">
+      <Text variant={headingSize} as="p" alignment="center">
+        {heading}
+      </Text>
+    </Box>
   ) : null;
 
   const childrenMarkup = children ? (
-    <div className={styles.Content}>
-      <Text as="span" variant="bodyMd" color="subdued">
+    <Box>
```

Comment:
I've been thinking about whether there's value in replacing a simple `div` with `Box` with no props. Curious to hear your thoughts.